python/lsst/analysis/ap/nb_utils.py

```diff
@@ -263,31 +263,33 @@ def compare_sources(butler1, butler2, query1, query2,
         for i in range(len(unique1)):
             dId = unique1.iloc[i]['diaSourceId']
             idx = unique1.index[i]
-            unique1.at[idx, 'pathexists'] = os.path.exists(cpath1.call(dId))
+            unique1.at[idx, 'pathexists'] = os.path.exists(cpath1(dId))
         pathchk1 = unique1.loc[~unique1['pathexists']]
 
         unique2['pathexists'] = False
         for i in range(len(unique2)):
             dId = unique2.iloc[i]['diaSourceId']
             idx = unique2.index[i]
-            unique2.at[idx, 'pathexists'] = os.path.exists(cpath2.call(dId))
+            unique2.at[idx, 'pathexists'] = os.path.exists(cpath2(dId))
         pathchk2 = unique2.loc[~unique2['pathexists']]
 
         # Only write those that don't exist yet
         pisct.write_images(pathchk1, butler=butler1, njobs=njobs)
 
-        pisct.config.diff_image_type = diffimg_type2
-        pisct.config.output_path = cutout_path2
+        # Create a new instance for writing the second set of cutouts
+        config.diff_image_type = diffimg_type2
+        pisct = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask(
```

Comment:
Not to be too pedantic but I don't think this was actually a bug since it should be enough to change the path, though I agree it is clearer what's happening in this new way.

Reply:
The problem is that the property is called `_output_path`, so we'd have to change that to `pisct.config._output_path`, and I didn't like the look of setting what looks like a private property.